src/card/price.rs

```diff
@@ -2,7 +2,7 @@
 use serde::{Deserialize, Serialize};
 
 /// Struct defining a price object containing data in various currencies.
-#[derive(Serialize, Deserialize, Clone, Eq, PartialEq, Ord, PartialOrd, Hash, Debug, Default)]
```

Comment:
hmm.. this one is tricky no? Sorting by price might make sense. Although from the contents of the struct I don't know how useful the derived ordering will/could be.

Reply:
We can't do price conversions, especially between real currencies and Tix, so I removed this one.

Reply:
I suppose I could write a bespoke comparison, comparing each field with the corresponding field from `other`, and returning `Some(ordering)` if they all match and `None` if they don't.

Reply:
Price comparisons between anything and Black Lotus would always return `None`, and comparisons between anything and Forest would almost always return `Some(..)`.